spec/policies/organisation_policy_spec.rb

```diff
@@ -19,15 +19,14 @@
       expect(subject).to permit(create(:user_in_organisation, role: 'admin'), build(:organisation))
     end
 
-    it "allows organisation admins to assign a user only to organisations within their organisation subtree" do
+    it "is forbidden for organisation admins even when assigning a user to organisations within their organisation subtree" do
```

Comment:
Thanks for picking up on it.  On reflection maybe the suffix isn't needed at all.  Just "is forbidden for organisation admins" feels like it's enough.  The details of "with their org" or "an org in their subtree" can live in the spec itself (and are really only relevant when we think about the permissions org admins *used* to have.